docs/build-on-linea/tooling/attestations/attestations.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Attestations
```

Comment:
This is minor, but is it best practice in these definitions to have the title be singular, rather than plural?

Reply:
I am not sure. In this case I just have the title match the sidebar name so that it is consistent

Reply:
got it